test/utils.test.ts

```diff
@@ -186,4 +187,109 @@ describe('Utils', () => {
       });
     });
   });
+
+  describe('processConfig', () => {
+    it('Should use a valid config', async () => {
+      fs.readFile = jest.fn().mockResolvedValueOnce(
+        JSON.stringify({
+          include: './contracts/**/*.sol',
+        })
+      );
+      const config = await utils.processConfig(path.join(__dirname, './valid.config.json'));
+
+      // The default settings should be applied
+      expect(config).toEqual({
+        root: './',
+        include: './contracts/**/*.sol',
+        exclude: '',
+        enforceInheritdoc: true,
+        constructorNatspec: false,
+      });
+    });
+
+    it('Should revert with an invalid config', async () => {
```

Comment:
Lowercase to match the rest of the tests 😬
```suggestion
    it('should revert with an invalid config', async () => {
```